src/keywords.js

```diff
@@ -10,6 +10,13 @@ export syntax galat = function (ctx) {
     return #`false`;
 }
 
+
+// isNaN
+export syntax kya_galat = function (ctx) {
+    let params = ctx.next().value
+    return #`isNaN ${params}`;
+}
+
 // if
```

Comment:
since we are inheriting from JavaScript, we should rather use camelCase than snake_case.
secondly isNaN => (isNotANumber()), so kyaNumberNhi **kNN** makes more sense than knH
k should be small so as to know its a method, not a class.